### DISCUSSION TITLE
[Detect:Warning]	(Another tablet driver found: Wacom) while using XP Pen Artist 12

### DISCUSSION THREAD

**Discussion by aiicarus** [OpenTabletDriver/OpenTabletDriver#2603] [Q&A] [answered]
Im using an XP Pen Artist 12 and trying to use OpenTabletDrivers but I get this error and cursor flickering between two spots on the screen. I've never installed a wacom driver or used a wacom device on this computer before so I'm not sure what drivers I'm supposed to uninstall to make this work as I've already uninstalled the official XP Pen drivers. This helped initially but the next time I used the tablet I got the same error.

**Comment by jamesbt365** [comment, marked as answer]
https://github.com/X9VoiD/TabletDriverCleanup/releases/tag/v4.0.2

**Comment by gonX** [comment]
Please see https://opentabletdriver.net/Wiki/FAQ/Windows#troubleshooting if it still doesn't work.
